packages/playground/remote/src/lib/offline-mode-cache.ts

```diff
@@ -23,21 +23,13 @@ export class OfflineModeCache {
 		this.cache = cache;
 	}
 
-	async removeOutdatedFiles() {
-		const keys = await caches.keys();
-		const oldKeys = keys.filter(
-			(key) =>
-				key.startsWith(CACHE_NAME_PREFIX) && key !== LATEST_CACHE_NAME
-		);
-		return Promise.all(oldKeys.map((key) => caches.delete(key)));
-	}
-
 	async cachedFetch(request: Request): Promise<Response> {
 		if (!this.shouldCacheUrl(new URL(request.url))) {
 			return await fetch(request);
 		}
 
 		let response = await this.cache.match(request, { ignoreSearch: true });
+		// @TODO: this.cache.match() returns stale data even after the cache has been purged
```

Comment:
Is this something you've observed in testing?

Reply:
Yeah but that was because of stale HTTP cache, not because of the `match` call

Reply:
> Yeah but that was because the stale HTTP cache caused the old assets to be cached again under a fresh key. The match() calls seems fine.

Ah, that makes sense. Good to hear. The TODO is now removed.